### DISCUSSION TITLE
Host config update - config in container is updating but server does not detect changes

### DISCUSSION THREAD

**Discussion by ikcehcus** [bluenviron/mediamtx#672] [Questions]
### Which version are you using?
latest

### Which operating system are you using?
OS

- [x]  Linux
- [x] Windows

### Describe the issue

I run container:

`docker run --rm -it -e RTSP_PROTOCOLS=tcp -p 8554:8554 -p 1935:1935 -v C:\rtsp-simple-server-config:/config aler9/rtsp-simple-server /config/rtsp-simple-server.yml`

Next, I change rtsp-simple-server.yml in host (C:\rtsp-simple-server-config)

No logs about config reload

When I do

`docker cp containerid:/config .`

I can see that content of the config in the container was changed so the file was updated after the change on the host.

**edit:**

I also tried on Linux (as host) but the behavior is the same.

**Comment by aler9** [comment]
Hello,

Regarding Windows and Docker, assuming that the backend is WSL2, the [Docker documentation](https://docs.docker.com/desktop/windows/wsl/) explicitly says that <i>Linux containers only receive file change events (“inotify events”) if the original files are stored in the Linux filesystem</i>. Therefore, you have to store the configuration file on the Linux filesystem.

Regarding Linux and Docker, this works flawlessly:

```
echo "paths:" > rtsp-simple-server.yml
docker run --rm -e RTSP_PROTOCOLS=tcp -p 8554:8554 -p 1935:1935 -v $PWD:/config aler9/rtsp-simple-server /config/rtsp-simple-server &
echo "  all:" >> rtsp-simple.server.yml
```

resulting in:

```
2021/10/29 10:12:00 INF [0/0] reloading configuration (file changed)
```

Finally, if you want a safer approach to edit the configuration, take a loot at the [HTTP API](https://github.com/aler9/rtsp-simple-server#http-api).

[a comment by an automated account is not shown]
